packages/lib-core/src/runtime/PluginStore.ts

```diff
@@ -54,6 +55,8 @@ export class PluginStore implements PluginStoreInterface {
   /** Feature flags used to determine the availability of extensions. */
   private featureFlags: FeatureFlags = {};
 
+  readonly sdkVersion = sdkVersion;
```

Comment:
@vojtechszocs I am wondering if we need to go through all the effort of exposing the version or if we should just log it to the console ourselves. We already log info to the console about which plugins are loading etc.

Reply:
Sorry for responding late. I think it's best to leave this to the host application, with us providing the actual version via the `sdkVersion` property. Plugin runtime infra init code is a responsibility of the host application, logging the version should be part of that code.